owasp-suppressions.xml

```diff
@@ -16,4 +16,11 @@
     <packageUrl regex="true">^pkg:maven/io\.github\.graphql\-java/graphql\-java\-annotations@.*$</packageUrl>
     <cpe>cpe:/a:graphql-java:graphql-java</cpe>
   </suppress>
+  <suppress until="2023-06-29Z">
```

Comment:
Adding a suppression with expiry for now as there is no update available yet and the authenticity if the vulnerability is being disputed - https://github.com/FasterXML/jackson-databind/issues/3973.